webpack.config.js

```diff
@@ -15,10 +14,18 @@ module.exports = {
             loader: 'babel-loader',
         }],
     },
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+        }),
+    ].concat(production ?  [
+        new webpack.optimize.UglifyJsPlugin(),
```

Comment:
@jpetitcolas: Is it better to use command line arguments or explicit configuration ? It does the same thing but the explicit configuration allow you to specify options to plugins (and you have only one file to read to see what it's done by running simple `webpack` command).